internal/index_read.go

```diff
@@ -102,26 +103,19 @@ func readBins(r io.Reader, typ string) ([]Bin, *ReferenceStats, error) {
 	return bins, stats, nil
 }
 
-func readChunks(r io.Reader, n int32, typ string) ([]bgzf.Chunk, error) {
+func readChunks(r io.Reader, n int32, typ string, buf [16]byte) ([]bgzf.Chunk, error) {
```

Comment:
I don't understand this. You are stack allocating a `[16]byte` for each call by this, why not just have `var buf [16]byte` prior to the loop?